SYCL/SubGroup/generic-shuffle_aspect-fp64.cpp

```diff
@@ -0,0 +1,21 @@
+// Enable FP64 part of <generic-shuffle.cpp>. To be removed once DPC++
+// supports optional device features and the code could be enabled
+// unconditionally without causing failures in speculative compilation
+// of the kernels.
+//
+// REQUIRES: aspect-fp64
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -DENABLE_FP64 %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//
+//==-- generic_shuffle_aspect-fp64.cpp - SYCL sub_group generic shuffle test *-
+// C++ -*--==//
```

Comment:
This looks wrong. Can you please fix such dangling formatting throughout the whole PR?